tests/unit/ui/components/TestRating.js

```diff
@@ -238,14 +283,26 @@ describe(__filename, () => {
       sinon.assert.notCalled(onSelectRating);
     });
 
+    it('does nothing when you hover over stars', () => {
+      const root = render({ readOnly: true });
+
+      const rating = 3;
+      const hoverStar = getStar({ root, rating });
+      hoverStar.simulate('mouseEnter', createFakeEvent());
+
+      expect(getStar({ root, rating })).not.toHaveClassName(
```

Comment:
This test is checking for a specific behaviour being absent, but the test says "does nothing when you hover over stars", so would it be any better to check that the classNames of the star are the same before and after simulating the hover, rather than checking for the absence of this specific class?